courses/free-courses-ar.md

```diff
@@ -349,20 +366,43 @@
+* [دورة Laravel 6/7/8 PHP&rlm;](https://www.youtube.com/playlist?list=PLMYF6NkLrdN8V2JKIMxqMsZNPsgUj3WOK) - Muhammed Essa
+* [كورس اساسيات لارافيل 9](https://www.youtube.com/playlist?list=PLftLUHfDSiZ4GfPZxaFDsA7ejUzD7SpWa) - Mora Soft
+* [مشروع لارافل متجر الكتروني متعدد التجار واللغات متكامل - تطبيقات لارافل عمليه](https://www.youtube.com/playlist?list=PLCm7ZeRfGSP7DCb-NamG2tt7uQUfxP2va) - Professional Code
+* [API-Course-For-Beginners&rlm;](https://www.youtube.com/playlist?list=PLftLUHfDSiZ6MfN8UhhcXDhh64eejvIKK) - Mora Soft
+* [Build and deploy laravel realtime application on AWS for FREE (Arabic)&rlm;](https://www.youtube.com/playlist?list=PL7IXur3gcVAT4wUbA3KpOSSDm3j_n-qw5) - Ibrahim Konsowa
+* [Laravel 8 Tutorial - دورة لارافيل 8 باللغة العربية من الصفر الى الاحتراف](https://www.youtube.com/playlist?list=PLd4ZH7drWj7DAt5osYlsya3sscPoERtGC) - Mohammed Mustafa
+* [Laravel 8 payment gateway - الدفع الالكتروني باستخدام لارافل 8 - الدفع البنكي بلارافل](https://www.youtube.com/playlist?list=PLCm7ZeRfGSP4wtrnHHJyf8a7E26I3ZrPG) - Professional Code
+* [Laravel API Complete Tutorial -شرح laravel API- شرح API لارافل - كورس - شرح laravel api باملثة عملية](https://www.youtube.com/playlist?list=PLCm7ZeRfGSP5e07XG-waxCb2kLq7M4J5m) - Professional Code
+* [Laravel in Arabic Framework 2020 - شرح لارافل دوره لارافل بالعربي المتكاملة بمشروع متجر](https://www.youtube.com/playlist?list=PLCm7ZeRfGSP4NNEikwx3wUAskQHB3p-LK) - Professional Code
+* [laravel + Vue.js complete tutorial - لارافل وفيو بمشروع تطبيق متكامل شرح](https://www.youtube.com/playlist?list=PLCm7ZeRfGSP5NRdNtBfznKtFoEAuKEd2n) - Professional Code
+* [larave advanced - realtime notification - laravel chat - laravel firebse اشعارات الزمن الحقيقي و فاير بيز لارافل](https://www.youtube.com/playlist?list=PLCm7ZeRfGSP5u3WilkVBz4s-uQtsy79eb) - Professional Code
+* [laravel websockets tutorial realtime notifications - chatting webscokets with node.js without any price&rlm;](https://www.youtube.com/playlist?list=PLCm7ZeRfGSP43uz7u3096X9QY4Gs_oAFt) - Professional Code
+* [Mastering Laravel&rlm;](https://www.youtube.com/playlist?list=PLDoPjvoNmBAy_mAhY0x8cHf8oSGPKsEKP) - Elzero Web School
+* [Redis + Laravel 8 complete tutorial - لارافل ريدس _ Redis queue with laravel- Redis caching with laravel&rlm;](https://www.youtube.com/playlist?list=PLCm7ZeRfGSP5CVYv0ABdApuYekWKcPNIT) - Professional Code
+
+
 ### Prolog
 
 * [Logic Programming - Prolog - برمجة منطقية](https://www.youtube.com/playlist?list=PLMm8EjqH1EFW9Faldu6D6Uh2j1EWWaTYe) - Ahmed Sallam
 
 
 ### Python
 
-* [كورس أساسيات المفاهيم البرمجية](https://www.youtube.com/playlist?list=PLvGNfY-tFUN9OQ4xtJRutrX3zyGam6Vpm) - غريب الشيخ \|\| Ghareeb Elshaikh&rlm; (:construction: *in process*)
+* [كورس تعلم أساسيات البرمجة للمبتدئين كامل \|\| محو الأمية البرمجية](https://www.youtube.com/watch?v=06h470AiBZ4) - غريب الشيخ \|\| Ghareeb Elshaik
```

Comment:
He deleted the last playlist and combined it into one video, so I just updated it and put that video, can we put postfix text (Combined Video) like this

```
* [name](url) - name (Combined Video)
```

Reply:
He just created new playlist for it, so I update it again now